pandas/tests/conftest.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+import numpy as np
+
+from pandas import Series
+import pandas.util.testing as tm
+
+
+@pytest.fixture(params=[tm.makeBoolIndex(10, name='a'),
+                        tm.makeIntIndex(10, name='a'),
+                        tm.makeFloatIndex(10, name='a'),
+                        tm.makeDateIndex(10, name='a'),
+                        tm.makeDateIndex(10, name='a').tz_localize(
+                            tz='US/Eastern'),
+                        tm.makePeriodIndex(10, name='a'),
+                        tm.makeStringIndex(10, name='a'),
+                        tm.makeUnicodeIndex(10, name='a')])
+def indices(request):
+    return request.param
+
+
+arr = np.random.randn(10)
+
+
+@pytest.fixture(params=[Series(arr, index=tm.makeBoolIndex(10, name='a')),
+                        Series(arr, index=tm.makeIntIndex(10, name='a')),
+                        Series(arr, index=tm.makeFloatIndex(10, name='a')),
+                        Series(arr, index=tm.makeDateIndex(10, name='a')),
+                        tm.makeDateIndex(10, name='a').tz_localize(
+                            tz='US/Eastern').to_series(keep_tz=True),
+                        Series(arr, index=tm.makePeriodIndex(10, name='a')),
+                        Series(arr, index=tm.makeStringIndex(10, name='a')),
+                        Series(arr, index=tm.makeUnicodeIndex(10, name='a'))])
+def series(request):
```

Comment:
need to rename to series_fixture; add a doc-string as well

Reply:
renamed and added doc string